src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,18 @@
+package com.internet.shop.model;
+
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class Order {
+    private static long idCount = 1;
```

Comment:
Of course, this logic will work to, but in task we were asked to make increment in Storage...